web/src/components/common.js

```diff
@@ -5,6 +5,27 @@ import Col from 'react-bootstrap/Col'
 import Card from 'react-bootstrap/Card'
 
 
+function change_url(x, name=null) {
```

Comment:
We should probably add the review system base url in each Change and Change events in the DB. And makes the web UI able to react on that. We should be able to handle any gerrit instances as well and also any github enterprise deployments.

Reply:
sure it would be cleaner

Reply:
@morucci where should I do that in the code?

Reply:
I tried to do it for github pr. let me know if it was the right way